modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -22,7 +22,7 @@ Remote database aliases require configuration to safely connect to the remote ta
 It is not possible to impersonate a user on the remote database or to execute an administration command on the remote database via a remote database alias.
 
 Database aliases can be created and managed using a set of Cypher administration commands executed against the `system` database.
-The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[Access control -> DBMS administration].
```